virtcontainers/pkg/annotations/annotations.go

```diff
@@ -44,6 +44,9 @@ const (
 	// JailerPath is a sandbox annotation for passing a per container path pointing at the jailer that will constrain the container VM.
 	JailerPath = kataAnnotHypervisorPrefix + "jailer_path"
 
+	// CtlPath is a sandbox annotation for passing a per container path pointing at the acrn ctl binary
+	CtlPath = kataAnnotHypervisorPrefix + "ctlpath"
```

Comment:
Ah, see that you added this as an annotation. I would rather go in the other direction of trimming down our annotations in the future and only support those that are truly required, rather than support all configs as annotations from usability and maintenance point of view . Would like to see what others think about this.
Also, since this is a very `ACRN` specific annotation, I am not in favor of adding this unless someone explicitly requests for it to be added.

Reply:
This is a security risk. It is important to ensure that if the annotation is ever added, it follows the correct pattern. Otherwise, we reopen the big hole that just series closes.

Reply:
@c3d, we're on the same page on how important it's to ensure that if an annotation is ever added, it must follow the correct pattern. However, I also agree with @amshinde that we should not add any new annotation unless it's stirctly needed. With this in mind, I'd like to understand how CtlPath would take advantage of an annotation.

@kata-containers/runtime, any opinion here?